self_supervised_pretraining/swinunetr_pretrained/README.md

```diff
@@ -0,0 +1,78 @@
+# Pre-trained SwinUNETR Backbone on ~50,000 3D Volumes
+
+This tutorial is an example of how to use the latest set of SwinUNETR pre-trained weights. The pre-training was done using multiple public datasets with over ~58,000 3D CT and MR volumes.
+
+Benchmark on BTCV with using the pre-trained weights as compared to random initialization shows promising gains in terms of accuracy/performance.
+
+### Steps to run the tutorial
+1. Download the dataset [BTCV](https://www.synapse.org/#!Synapse:syn3193805/wiki/217789) (More detail is provided in the Data section)
+2. Download the pretrained SwinUNETR weights on the 50,000 3D volumes from the following link. More information about the dataset and training is provided below
```

Comment:
link is missing here.

Reply:
https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1 now has the "ssl_pretrained_weights.pth" [here](https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/ssl_pretrained_weights.pth)

Please update and verify if the file upload is correct. Thanks!

Reply:
I have updated the link, thank you. Verification in process.